test/support/example_behaviour.ex

```diff
@@ -221,14 +221,14 @@ end
 
 defmodule ElixirSenseExample.ExampleBehaviourWithDocCallback do
   @behaviour ElixirSenseExample.ExampleBehaviourWithDoc
-  @behaviour GenServer
-  @behaviour :gen_statem
 
+  @impl true
```

Comment:
`@impl` is missing here intentionally, see https://github.com/elixir-lsp/elixir_sense/pull/84/commits/ada2907467a3b7bdb9d7fd55309e58e398849208

Reply:
Would you please be so kind as to help me with that? Feel free to add commits to the branch.

Reply:
Done on master

Reply:
Thanks, @lukaszsamson